creator-node/src/components/healthCheck/healthCheckController.js

```diff
@@ -92,17 +90,16 @@ const healthCheckVerboseController = async (req) => {
 }
 
 /**
- * Controller for `health_check/fileupload` route
- * Calls `healthCheckFileUploadService`.
+ * Controller for `health_check/fileupload` route *
+ * Perform a file upload health check limited to configured delegateOwnerWallet.
+ * This prunes the disc artifacts created by the process after.
  */
 const healthCheckFileUploadController = async (req) => {
-  const err = req.fileFilterError || req.fileSizeError
+  const err = req.fileFilterError || req.fileSizeError || await removeTrackFolder(req, req.fileDir)
   if (err) {
     throw new Error(err)
-  } else {
-    removeTrackFolder(req, req.fileDir)
-    return successResponse({ success: true })
   }
+  return successResponse({ success: true })
```

Comment:
its inconsistent here to throw an Error and return successResponse, the right pattern is to use the error api helpers and instead of throwing an error call return ErrorResponseServerError